<!doctype html><html prefix="og: http://ogp.me/ns#"><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><title>Postdoc &#183; Cody Markelz</title><meta name=description content="--NSF Postdoctoral Research Fellowship Projects"><meta name=HandheldFriendly content="True"><meta name=MobileOptimized content="320"><meta name=generator content="Hugo 0.77.0"><meta name=robots content="index,follow"><meta name=viewport content="width=device-width,initial-scale=1"><meta property="og:title" content="Postdoc"><meta property="og:description" content="--NSF Postdoctoral Research Fellowship Projects"><meta property="og:type" content="article"><meta property="og:url" content="https://rjcmarkelz.github.io/project/postdoc_projects/"><link rel=stylesheet href=https://rjcmarkelz.github.io/dist/site.css><link rel=stylesheet href=https://rjcmarkelz.github.io/dist/syntax.css><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,400,600,700,300&subset=latin,cyrillic-ext,latin-ext,cyrillic"><link rel=stylesheet href=https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css integrity=sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN crossorigin=anonymous></head><body><div id=wrapper><header class=site-header><div class=container><div class=site-title-wrapper><h1 class=site-title><a href=https://rjcmarkelz.github.io/>Cody Markelz</a></h1><a class="button-square button-social hint--top" data-hint=Twitter aria-label=Twitter href=https://twitter.com/rjcmarkelz rel=me><i class="fa fa-twitter" aria-hidden=true></i></a><a class="button-square button-social hint--top" data-hint=Github aria-label=Github href=https://github.com/rjcmarkelz rel=me><i class="fa fa-github-alt" aria-hidden=true></i></a><a class="button-square button-social hint--top" data-hint=LinkedIn aria-label=LinkedIn href=https://www.linkedin.com/in/codymarkelz rel=me><i class="fa fa-linkedin" aria-hidden=true></i></a><a class="button-square button-social hint--top" data-hint=Email aria-label="Send an Email" href=mailto:rjcmarkelz@berkeley.edu><i class="fa fa-envelope" aria-hidden=true></i></a></div><ul class=site-nav><li class=site-nav-item><a href=/>Blog</a></li><li class=site-nav-item><a href=/project/>Projects</a></li><li class=site-nav-item><a href=/page/publications/>Publications</a></li><li class=site-nav-item><a href=/page/cv/>CV</a></li><li class=site-nav-item><a href=/page/about/>About</a></li></ul></div></header><div id=container><style>.site-title a,.button-square{background:#000}a{color:#000}</style><div class=container><article class=post-container><header class=post-header><h1 class=post-title>Postdoc</h1><p class=post-description>--NSF Postdoctoral Research Fellowship Projects</p></header><div class="post-content clearfix"><h3 id=physiological-systems-biology-of-competition-in-brassica-rapa>Physiological Systems Biology of Competition in <em>Brassica rapa</em></h3><p>Plants growing in dense stands compete with one another for light resources at the top of the canopy. When plants sense neighbors through the phytochrome photoreceptor system they become apical dominant and grow upwards to maximize light capture. This response is termed shade avoidance and has very important ecological and agronomic consequences if plant resources are used for competitive growth instead of reproductive output. For example, over-seeding in an agricultural field is wasteful of seed resources if many individuals die after competing with neighbors for limited light and nutrient resources. In ecological settings, mixed plant communities are competing for resources in a similar way to agricultural settings, but without the direct human management of inputs. In order to study the agronomic and ecological consequences of competition I am worked with the emerging model species <em>Brassica rapa</em>. <em>B. rapa</em> is an ideal model for plant competition research because it has a sequenced genome, is both a crop and a weed in agricultural settings, and is a highly morphologically and physiologically diverse species. I am approached this from many different angles that relied on my diverse research background and interests.</p><p><img src=/static/img/research_overview.jpg alt=research-overview></p><h3 id=1-b-rapa-trait-database>1) <em>B. rapa</em> trait database</h3><p><img src=/static/img/graph_db.jpg alt=graph_db></p><p>Hard won biological data (especially from the field) should not just sit in a lab note book or on spread sheets scattered across various machines. Sharing phenotypic data after publication should be a priority for us as a community, just like putting sequencing data into NCBI databases (<a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001595"><strong>see Zamir 2013</strong></a>). This allows for the data to be used to ask multiple questions, for modeling, or to be analyzed using systems biology techniques. Towards all these goals I designed a graph database to house all of the published trait data, experimental meta-data, publications, gene expression data, from the <em>B. rapa</em> population.</p><p>A partial realization of this work has come out of working with an undergraduate <a href=https://github.com/tiaho>Tiffany Ho</a> to create an interactive data visualization tool to examine co-occurrence of eQTL with physiological QTL called <a href=http://malooflab.phytonetworks.org/apps/qtl-visualization/><strong>QTLVisR</strong></a>.</p><h3 id=2-statistical-genetics>2) Statistical Genetics</h3><p>In collaboration with <a href=http://mfcovington.github.io/><strong>Mike Covington</strong></a> (who generated the SNPs), I created a saturated genetic map for this population (<a href=https://github.com/rjcmarkelz/brassica_genetic_map_paper><strong>here</strong></a>). This new map, along with a Bayesian statistical approach and my graph database all blends together into a fun data science project. I am remapping all the published traits in the database using a multi-trait approach. This data is used directly in the next two aspects of my project.</p><p><a href=/static/pdfs/Markelz_etal_2017.pdf><strong>Markelz et al. 2017</strong></a><br><a href=/static/pdfs/Brock_etal_2016.pdf><strong>Brock et al. 2016</strong></a></p><p><img src=/static/img/genetic_map.jpg alt=genetic_map></p><h3 id=3-multi-scale-data-integration>3) Multi-Scale Data Integration</h3><p>The central dogma of molecular biology says DNA → RNA → Protein. Scaling the actions and interactions of proteins spatially across tissue or cell types is physiology. Scaling the actions of physiology across space and time is development. Updating the simple model to include physiology and development would look something like this: DNA → RNA → Protein → Metabolites → Physiology → Development. Of course there are feedbacks every step of the way, and it is hard to disentangle each of these into separate processes, but this simple conceptual model goes a long way. If there is a genomic location for these interactions explaining some of the variance observed in the physiological or developmental phenotype, then there is a QTL in that genomic location. QTL studies are conducted using genetic mapping populations where the parents of these populations are differ significantly from one another in the trait of interest. Up until recently, a majority of QTL studies have focused on traits that are the result of many proteins interacting one level of biological organization below the physiological or developmental trait. It is a large jump in biological organization to work backwards from physiology and development to DNA sequence differences. Quantifying messenger RNA (mRNA) acts as a stepping stone between the physiology and the DNA. Or to put it another way, mRNA can provide clues as to what proteins cells are planning to make given the current information from internal developmental and external environmental cues. The collection of mRNA expression is generally referred to as the gene co-expression network. I am interested in understanding the gene co-expression network and how it relates to DNA polymorphisms in one direction and physiological and developmental phenotypes in the other direction. In order to be able to do this, we not only need a good understanding of the biology, but we also need a good understanding of the ways in which these pieces of biological information fit together. I developed statistical techniques to connect these pieces of information in a probabilistic framework.</p><p><a href=/static/pdfs/Baker_etal_2019.pdf><strong>Baker et al. 2019</strong></a>
<a href=/static/pdfs/Markelz_etal_2017.pdf><strong>Markelz et al. 2017</strong></a><br><a href=/static/pdfs/Brock_etal_2016.pdf><strong>Brock et al. 2016</strong></a><br><a href=/static/pdfs/Baker_etal_2015.pdf><strong>Baker et al. 2015</strong></a></p><h3 id=4-genetically-informed-b-rapa-physiological-growth-models>4) Genetically Informed <em>B. rapa</em> Physiological Growth Models</h3><p>I am using this data to parameterize physiologically based models of plant growth. Coming full circle to my PhD work in metabolism and physiology. This is my favorite part of my project. It involves integrating environmental time course data, physiological QTLs, and programming mathematical models of plant growth. Based on my meta-analysis results, and preliminary physiological modeling results, I choose a subset of genotypes to grow in the field for 2014 to test model predictions. The field site I work at is home to three close collaborators at University of Wyoming: <a href=http://www.robertlbaker.org/Evolution_and_Development.html>Dr. Rob Baker</a>, Dr. Marc Brock, and <a href=http://www.uwyo.edu/molecbio/faculty-and-staff/cyntia-weinig.html>Dr. Cynthia Weinig</a>.</p><p><a href=/static/pdfs/Baker_etal_2019.pdf><strong>Baker et al. 2019</strong></a><br><a href=/static/pdfs/Baker_etal_2015.pdf><strong>Baker et al. 2015</strong></a></p><p>Here is a preview of a canopy scale model output predicting peak canopy N content for four different treatments. Notice that there is a clear interaction between N availability and crowding as to when there is peak canopy N.</p><p><img src=/static/img/physiology_simulations.jpg alt=physiology_simulations></p><h3 id=5-computer-vision-plant-quantification-and-3d-reconstruction>5) Computer Vision Plant Quantification and 3D Reconstruction</h3><p>I developed a high-throughput 3D imaging robot and analysis pipeline to take non-destructive plant growth data for model calibration/validation of <em>Brassica rapa</em>. It was really enjoyable to build equipment. It pulls together many ways of thinking about biological problems. It forces the biologist to really think about what process they are trying understand and how to quantify it using some sort of sensor. These projects help biologists build intuition about measurement error, instrument limitation, methods development, materials selection, and design. These are skills that are undervalued in biology.</p><p><a href=/static/pdfs/Bucksch_etal_2017.pdf><strong>Bucksch et al. 2017</strong></a><br><a href=/static/pdfs/Friesner_etal_2017.pdf><strong>Friesner et al. 2017</strong></a></p><p>I also co-designed a similar system for measuring <em>Arabidopsis</em> growth.</p><p><a href=/static/pdfs/An_etal_2016.pdf><strong>An et al. 2016</strong></a><br><a href=/static/pdfs/An_etal_2017.pdf><strong>An et al. 2017</strong></a></p><p><img src=/static/img/3D_summary.jpg alt=3D_summary></p><div style=position:relative;padding-bottom:56.25%;height:0;overflow:hidden><iframe src=https://player.vimeo.com/video/108757972 style=position:absolute;top:0;left:0;width:100%;height:100%;border:0 title="vimeo video" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe></div></div><footer class="post-footer clearfix"><div class=share><a class=icon-twitter href="https://twitter.com/share?text=Postdoc%20%20&url=https%3a%2f%2frjcmarkelz.github.io%2fproject%2fpostdoc_projects%2f" onclick="window.open(this.href,'twitter-share','width=550,height=235');return false;" aria-label="Share on Twitter"><i class="fa fa-twitter" aria-hidden=true></i></a><a class=icon-facebook href="https://www.facebook.com/sharer/sharer.php?u=https%3a%2f%2frjcmarkelz.github.io%2fproject%2fpostdoc_projects%2f" onclick="window.open(this.href,'facebook-share','width=580,height=296');return false;" aria-label="Share on Facebook"><i class="fa fa-facebook" aria-hidden=true></i></a></div></footer></article></div></div></div><footer class=footer><div class=container><div class=site-title-wrapper><h1 class=site-title><a href=https://rjcmarkelz.github.io/>Cody Markelz</a></h1><a class="button-square button-jump-top js-jump-top" href=# aria-label="Back to Top"><i class="fa fa-angle-up" aria-hidden=true></i></a></div><p class=footer-copyright><span>&copy; 2021 / Powered by <a href=https://gohugo.io/>Hugo</a></span></p><p class=footer-copyright></p></div></footer><script src=https://rjcmarkelz.github.io/js/jquery-1.11.3.min.js></script><script src=https://rjcmarkelz.github.io/js/jquery.fitvids.js></script><script src=https://rjcmarkelz.github.io/js/scripts.js></script></body></html>